resources/views/layouts/app.blade.php

```diff
@@ -28,6 +28,7 @@
 			</div>
 		</div>
 		<div class="container">
+			<?php var_dump(Auth::user()) ?>
```

Comment:
Why is this here?

Reply:
Pa que sepan cómo verificar que haya un usuario.